lib/internal/blob.js

```diff
@@ -397,25 +397,27 @@ class Blob {
   }
 }
 
-function ClonedBlob() {
-  return ReflectConstruct(function() {
+class CloneableBlob extends Blob {
+  static {
```

Comment:
Is this correct?

Reply:
This is the only way that I found that correctly copy the content of the Blob.

The alternative ways using the prototype didn't work.